lib/fizzy/stack.hpp

```diff
@@ -6,31 +6,53 @@
 namespace fizzy
 {
 template <typename T>
-class Stack : public std::vector<T>
+class Stack
```

Comment:
This is based off evmone's because I was lazy to write the same for testing. Yes, it uses a fixed stack, just for conducting this simple test.